content/posts/roadmap.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "The future of Jellyfin"
+subtitle: "A look into what's to come"
+author: "Julien Machiels"
+githubusername: "MrTimscampi"
+date: ""
+justify: "center"
+---
+
+As we approach the end of a few big migrations, we thought it would be a good occasion to go over the next steps for Jellyfin, as well as answer some common questions.
+
+<!--more-->
+
+# Phase 1: A little catch-up
+
+Since the beginning of the project, we have had a few clear goals that, while never listed out right, were well known of the contributors. Here they are:
```

Comment:
```suggestion
Since the beginning of the project, we have had a few clear goals that, while never listed outright, were well known of the contributors. Here they are:
```